### PR TITLE
primary-site: default HTTP/HTTPS proxy egress to disabled

### PR DESCRIPTION
**Pull request by sofuture**
### Changelog
primary-site:
* fix: default HTTP/HTTPS proxy egress to disabled

### Docs
None

### Description
Default HTTP/HTTPS proxy egress to disabled.

### CODE REVIEW

File: charts/primary-site/values.yaml [hunk 1]
```diff
@@ -19,9 +19,9 @@ globals:
     storageProvider: google_cloud
     bucketName: foxglove-inbox
   
+  ## Optionally used to configure http/https proxy egress
```
Comment (defunctzombie) [inline charts/primary-site/values.yaml:22]:
```suggestion
  ## Configure an http/https egress proxy if your environment requires outgoing connections to go through a proxy. Set `enabled` to `true` and one of `httpProxy` or `httpsProxy`
```

I don't know if this wording is exactly right but I would put something like this. What is the purpose of `noProxy`?

Comment (sofuture) [reply charts/primary-site/values.yaml:22]: `NO_PROXY` is a list of hosts to not proxy for

Comment (sofuture) [reply charts/primary-site/values.yaml]: Updated

**Review by defunctzombie — APPROVED** [review]
Seems better - `noProxy` is weird to me and "appropriate values" is vague if we expect something specific (like comma separated lists for noProxy).

**Comment by sofuture** [comment]
The names of these proxy variables are all convention 🙃 There's no spec

**Pull request merged**